pages/agent/v3/elastic_ci_aws/managing_elastic_ci_stack.md

```diff
@@ -87,14 +87,14 @@ or a specific release available from the [releases page](https://github.com/buil
 The latest stable release can be deployed to any of our supported AWS Regions.
 
 The most recent build of the CloudFormation stack is published to
-`https://s3.amazonaws.com/buildkite-aws-stack/master/aws-stack.yml`, along with
+`https://s3.amazonaws.com/buildkite-aws-stack/main/aws-stack.yml`, along with
 a version for each commit at
-`https://s3.amazonaws.com/buildkite-aws-stack/master/${COMMIT}.aws-stack.yml`.
+`https://s3.amazonaws.com/buildkite-aws-stack/main/${COMMIT}.aws-stack.yml`.
```

Comment:
Not blocking: The formatting here also seems a bit awkward with these long URLs as part of the paragraph. I wonder if it would be a bit nicer to split them up, like:

````md
The most recent build of the CloudFormation stack is published to:

```text
https://s3.amazonaws.com/buildkite-aws-stack/main/aws-stack.yml
```

With a version for each commit also published at:

```text
https://s3.amazonaws.com/buildkite-aws-stack/main/${COMMIT}.aws-stack.yml
```
````

That makes them simpler to copy, too.